wfe/wfe.go

```diff
@@ -1852,27 +1868,37 @@ func (wfe *WebFrontEndImpl) updateChallenge(
 		return
 	}
 
-	var chalResp struct {
-		KeyAuthorization *string
-	}
-	err := json.Unmarshal(postData.body, &chalResp)
-	if err != nil {
+	// In strict mode we reject any challenge POST with a body other than `{}`.
+	// This matches RFC 8555 Section 7.5.1.
```

Comment:
This is true for all challenges Pebble implements, though [per this erratum](https://www.rfc-editor.org/errata/eid5729) it might not be true for future challenges. Since some people will probably turn to Pebble to understand the spec better, could you describe that situation and link to the erratum?

Reply:
Sure

Reply:
Resolved in 0580ec4